db/schema.rb

```diff
@@ -36,14 +36,6 @@
   end
 
   create_table "users", force: true do |t|
-    t.integer  "user_id"
-    t.string   "email"
-    t.string   "name"
```

Comment:
No, because it was added in another migration created above. I'm not too sure about this.